### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by terrainthesky-hub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,14 @@ languages/technologies you aren't familiar with is a skill that we want to
...
 
+
+
 The second goal is to start peeling away the layers of abstraction that students
 are used to from working with JavaScript and Python. Indeed, when you boil it
-all down, both JavaScript and Python are implemented in C in the first place! C
+all down, both JavaScript and Python are implemented in C in the first pl
+
+
+ace! C
```
Comment (michelangelo17) [inline README.md]: Return in middle of word. Please fix!

File: README.md [hunk 2]
```diff
@@ -11,9 +11,14 @@ languages/technologies you aren't familiar with is a skill that we want to
 cultivate in students during their time here at Lambda, which is why we expose
 students to both C and Python during their time here.
 
+
```
Comment (michelangelo17) [inline README.md:14]: Like the spacing!

**Review by michelangelo17 — CHANGES_REQUESTED** [review]
Please fix that one word problem!

File: README.md (hunk 1 above)
Comment (terrainthesky-hub) [reply README.md]: I'll fix it later

Comment (terrainthesky-hub) [reply README.md]: fixed it!

**Review by michelangelo17 — APPROVED** [review]

**Pull request merged**